### PR TITLE
Update E8EB11FFFE254CF9.json

### PR DESCRIPTION
**Pull request by Hitheshkaranth**

### CODE REVIEW

File: E8EB11FFFE254CF9.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
 		"gateway_ID": "E8EB11FFFE254CF9", 
 		"servers": [ 
 			{ 
-				"server_address": "router.eu.thethings.network", 
+				"server_address": "router.asia-se.thethings.network",
```
Comment (gonzalocasas) [inline E8EB11FFFE254CF9.json]: This is not a valid address. Check here for valid ones: https://www.thethingsnetwork.org/docs/gateways/packet-forwarder/semtech-udp.html#router-addresses

Comment (Hitheshkaranth) [reply E8EB11FFFE254CF9.json]: @gonzalocasas 
Does this mean I need to change the frequency at gateway registration upon changing it here? currently Im using India 865-867Mhz?

Comment (Hitheshkaranth) [reply E8EB11FFFE254CF9.json]: @gonzalocasas ,
Thanks for reply.
RAK831 is the board Im working on. @global_conf are you suggesting to change the frequency on radio0 and radio1?

Comment (gonzalocasas) [reply E8EB11FFFE254CF9.json]: Yes, but check this master files for how to change that exactly: https://github.com/TheThingsNetwork/gateway-conf

**Pull request merged**